chalicelib/checks/wrangler_checks.py

```diff
@@ -1318,6 +1320,13 @@ def check_for_ontology_updates(connection, **kwargs):
         # instead of repos etc, check download url for ontology header to get version
         elif o.get('download_url'):
             owl = requests.get(o['download_url'], headers={"Range": "bytes=0-2000"})
+            if owl.status_code == 404:
```

Comment:
Would it be worth it to have any retries here? Probably not as it's only happened the one time so far and we don't know how quickly it resolved.

Reply:
In the current case it was a permanent move, not a temp outage. I'll leave as is for now but can add retries if temporary 404s happen in the future.